### PR TITLE
Add a temporary home with write perm

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
Add a temporary home where we have write access to for verifying deps.

### CODE REVIEW

**Comment by ravisantoshgudimetla** [comment]
As of now, we are getting the following error when running verify deps:

`[ERROR]	Cache directory unavailable: mkdir /.glide: permission denied`

@rphillips @stevekuznetsov

File: ci-operator/config/openshift/cluster-kube-scheduler-operator/openshift-cluster-kube-scheduler-operator-master.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ tests:
     make verify-bindata
     make verify-bindata-v3.11.0
     make verify-generated
+    HOME=$( mktemp -d )
```
Comment (smarterclayton) [inline ci-operator/config/openshift/cluster-kube-scheduler-operator/openshift-cluster-kube-scheduler-operator-master.yaml]: just set `HOME=/tmp`

Comment (ravisantoshgudimetla) [reply ci-operator/config/openshift/cluster-kube-scheduler-operator/openshift-cluster-kube-scheduler-operator-master.yaml]: Done.

**Pull request merged**